src/Elastic.Apm.AspNetFullFramework/ElasticApmModule.cs

```diff
@@ -455,12 +405,24 @@ private static void SafeAgentSetup(string dbgInstanceName)
 			}
 			catch (Agent.InstanceAlreadyCreatedException ex)
 			{
-				BuildLogger().Scoped(dbgInstanceName)
-					.Error()
-					?.LogException(ex, "The Elastic APM agent was already initialized before call to"
-						+ $" {nameof(ElasticApmModule)}.{nameof(Init)} - {nameof(ElasticApmModule)} will use existing instance"
-						+ " even though it might lead to unexpected behavior"
-						+ " (for example agent using incorrect configuration source such as environment variables instead of Web.config).");
+				if (Agent.Components is FullFrameworkAgentComponents)
+				{
+					BuildLogger()
+						.Scoped(dbgInstanceName)
+						.Error()
+						?.LogException(ex, "The Elastic APM agent was already initialized before call to"
+							+ $" {nameof(ElasticApmModule)}.{nameof(Init)} - {nameof(ElasticApmModule)} will use existing instance");
+				}
+				else
+				{
+					BuildLogger()
+						.Scoped(dbgInstanceName)
+						.Error()
```

Comment:
As discussed over zoom, let's not consider this as an error - an info log would be better.